### ISSUE TITLE
Mystcraft Support

### ISSUE THREAD

**Issue by SirFell** [vorquel/SimpleSkyGrid#6]
This mod is not compatible with mystcraft (but compatible with rf tools).
If you try to create/login in world with sky grid generator when mystcraft is installed you'll get this crash:

```
java.lang.RuntimeException: Cannot Hotload Dim: Overworld is not Loaded!
    at net.minecraftforge.common.DimensionManager.initDimension(DimensionManager.java:230)
    at net.minecraft.server.MinecraftServer.func_71218_a(MinecraftServer.java:705)
    at Config.getWorldServer(Config.java:1746)
    at RandomMobs.entityLoaded(RandomMobs.java:49)
    at net.minecraft.client.renderer.RenderGlobal.func_72703_a(RenderGlobal.java:2892)
    at net.minecraft.world.World.func_72923_a(World.java:1347)
    at net.minecraft.client.multiplayer.WorldClient.func_72923_a(WorldClient.java:182)
    at net.minecraft.world.World.func_72838_d(World.java:1338)
    at net.minecraft.client.multiplayer.WorldClient.func_72838_d(WorldClient.java:159)
    at net.minecraft.client.multiplayer.WorldClient.func_73027_a(WorldClient.java:226)
    at net.minecraft.client.network.NetHandlerPlayClient.func_147281_a(NetHandlerPlayClient.java:771)
    at net.minecraft.network.play.server.S0FPacketSpawnMob.func_148833_a(SourceFile:97)
    at net.minecraft.network.play.server.S0FPacketSpawnMob.func_148833_a(SourceFile:15)
    at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
    at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2061)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

Modpack is the same as in #5 , just add mystcraft and mystcraft nei plugin to it. just in case: [mod list](http://pastebin.com/Gn1dgbgJ)
And full [crashlog](http://pastebin.com/UXFVD9Nm)

**Comment by Vorquel** [comment]
Thank you for submitting another bug. I will see what I can do.

**Comment by Vorquel** [comment]
I think I know what the problem is here. Unfortunately, I cannot get out a fix very soon. I need to learn something completely new to me in order to fix it. Thus, I will release a new build with just the fix for #5.
